dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsJdbcTask.java

```diff
@@ -48,6 +52,8 @@
 
 public class TeradataLogsJdbcTask extends AbstractJdbcTask<Summary> {
 
+  private static final int DBQLSQLTBL_SQLTEXTINFO_LENGTH = 31000;
```

Comment:
nit: I would prefer if this value is included in the range accepted from the user

Reply:
Done.